binary_transparency/firmware/devices/usb_armory/bootloader/main.go

```diff
@@ -54,24 +55,45 @@ func main() {
 		haltAndCatchFire(fmt.Sprintf("card detect error: %v\n", err), 2)
 	}
 
-	usbarmory.LED("white", true)
-
-	offset, err := strconv.ParseInt(Start, 10, 64)
+	kernelOffset, err := strconv.ParseInt(StartKernel, 10, 64)
 	if err != nil {
-		haltAndCatchFire(fmt.Sprintf("invalid start offset: %v\n", err), 3)
+		haltAndCatchFire(fmt.Sprintf("invalid kernel partition start offset: %v\n", err), 3)
+	}
+	proofOffset, err := strconv.ParseInt(StartProof, 10, 64)
+	if err != nil {
+		haltAndCatchFire(fmt.Sprintf("invalid proof partition start offset: %v\n", err), 3)
 	}
 
-	partition := &Partition{
+	kernelPart := &Partition{
 		Card:   card,
-		Offset: offset,
+		Offset: kernelOffset,
 	}
-
-	if err = conf.Read(partition, defaultConfigPath); err != nil {
+	if err = conf.Read(kernelPart, defaultConfigPath); err != nil {
 		haltAndCatchFire(fmt.Sprintf("invalid configuration: %v\n", err), 4)
 	}
 
+	usbarmory.LED("white", true)
+	h, err := hashPartition(89246720, kernelPart)
```

Comment:
That number is a bit magic. Should it be passed in as an argument? (well, initialized at compile time like StartKernel)

Reply:
Yeah, it _is_ a bit of a random number, but I've made it passable-in like the others for now.